source/System.h

```diff
@@ -181,6 +183,7 @@ class System {
 	std::vector<FleetProbability> fleets;
 	double habitable = 1000.;
 	double asteroidBelt = 1500.;
+	double border = 1000.;
```

Comment:
This should be `400.` not `1000.`

Reply:
1000 is the sweet spot where it's not a nuisance in systems that have little combat, but point-blank engagements are still usually avoided.
It's surprisingly close-range, you can't really have the border get any smaller before the currently-existent gameplay issues return.

Reply:
I would save changing the default value for every person who plays the game for its own discussion.

Reply:
This is a balance change so should get it's own PR separate from this under-the-hood change. Changing the default will make this a hard PR to merge.

Reply:
You can always edit and merge. But I strongly suggest you keep it at 1000, and then choose peaceful systems to have smaller borders. It's not an arbitrary number, it really is what's best.